apps/website/docs/react/api/create-dom-renderer.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 4
+---
+
+# createDOMRenderer
+
+`createDOMRenderer` is paired with `RendererProvider` component and is useful for [child window rendering](/react/guides/child-window-rendering) and [SSR](/react/guides/ssr-usage) scenarios. It helps to ensure that styles will be inserted to a proper document.
```

Comment:
should we mention that it' s possible to create custom renderers ? you can use the one in the playground as reference

Reply:
Not now probably as I am not entirely sure about the API. For example, what happens when we will have CSS extraction?